src/components/LoansByCategory/CategoryRowHover.vue

```diff
@@ -411,6 +431,27 @@ export default {
 		handleSetPreventUpdatingDetailedCard(newState) {
 			this.setPreventUpdatingDetailedCard(newState);
 		},
+		runningOnServer() {
```

Comment:
You should be able to just use `this.$isServer` to check this...

Reply:
Nice, that's better! Updated :+1: